examples/vmseries_ha/main.tf

```diff
@@ -35,17 +35,14 @@ resource "local_file" "bootstrap_xml" {
 resource "local_file" "init_cfg" {
 
   for_each = { for k, v in var.vmseries : k => v
-    if can(v.bootstrap_template_map)
+    if can(v.bootstrap_options)
```

Comment:
I'm confused about which approach are we using to initialize VM-Series in the example: using the init-cfg.txt and the bucket or using VM-Series VM metadata?

Reply:
Should we introduce some flag like `use_bootstrap_bucket` to select the bootstrap method?